lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscoverySpec.hs

```diff
@@ -293,7 +295,9 @@ prop_genChangeGap
 prop_genChangeGap g =
     property prop
   where
-    key = unsafeGenerateKeyFromSeed (mempty, mempty) mempty
+    Right phr =
+        fromText "abcdefghijklmnop" :: Either TextDecodingError (Passphrase "encryption")
+    key = unsafeGenerateKeyFromSeed (coerce phr, coerce phr) mempty
```

Comment:
The second coerce / passphrase seems wrong here. It's the encryption one and, it used to be empty. I believe the error you get from CI comes from here

Reply:
yes, I put mempty in generation passphrase